.env.example

```diff
@@ -1 +1,2 @@
 PORT=3000
+REDISTOGO_URL=redis://localhost:6379/
```

Comment:
That's correct. We serve the index page out of redis in production. In development, we run the ember-cli app on localhost:4200 and proxy api requests to localhost:3000/api. I might be nice to render some help text if you try to hit the index page in development and don't have redis configured. Without anything in redis it'll just render a blank page which is probably less helpful then having it blow up.